applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -1,11 +1,11 @@
-from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division  # makes KM backward compatible with python 2.6 and 2.7
 #import kratos core and applications
-import KratosMultiphysics
-import KratosMultiphysics.StructuralMechanicsApplication as StructuralMechanicsApplication
-import KratosMultiphysics.ContactStructuralMechanicsApplication as ContactStructuralMechanicsApplication
+import KratosMultiphysics as KM
```

Comment:
I think I like your new syntax
Avoids the long names and also the opportunity to use different abbreviations (e.g.
- `import KratosMultiphysics.StructuralMechanicsApplication`
- `import KratosMultiphysics.StructuralMechanicsApplication as KratosStructural`)
- ...